.circleci/test-deploy.yml

```diff
@@ -207,7 +207,11 @@ workflows:
             parameters:
               runner: [cimg, mac, alpine, windows]
       - orb-tools/pack:
-          filters: *filters
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+$/
```

Comment:
Given we use the tag twice, I would anchor/alias it like the other filters

Reply:
Updated the template to do what i mean here: https://github.com/CircleCI-Public/Orb-Template/pull/11/commits/7b37d65d8f9c8064302fc16e373160b3927d2094